core/src/main/java/hudson/cli/Connection.java

```diff
@@ -268,7 +268,7 @@ public PublicKey verifyIdentity(byte[] sharedSecret) throws IOException, General
 
             return spk;
         } catch (ClassNotFoundException e) {
-            throw new Error(e); // impossible
+            throw new LinkageError(e.getMessage(), e); // impossible
```

Comment:
Arguably should be `AssertionError` because we believe it should never happen.

Reply:
Agreed.